modules/home-manager/gtk.nix

```
@@ -121,10 +122,10 @@ in
     (mkIf enable {
       gtk.theme =
         let
-          gtkTweaks = concatStringsSep "," cfg.tweaks;
+          gtkTweaks = if cfg.tweaks == [ ] then "" else (concatStrings [ "+" (concatStringsSep "," cfg.tweaks) ]);
```

Apologies, I didn't format the suggestion...

```suggestion
          gtkTweaks =
            if cfg.tweaks == [ ] then
              ""
            else
              (concatStrings [
                "+"
                (concatStringsSep "," cfg.tweaks)
              ]);
```